lib/parse.js

```diff
@@ -379,8 +379,12 @@ function tokenizer($TEXT, filename, html5_comments, shebang) {
                 return "";
             }
         }
-        if (ch >= "0" && ch <= "7")
-            return read_octal_escape_sequence(ch);
+        if (ch >= "0" && ch <= "7") {
+            if (next_token.has_directive("use strict"))
+                return read_strict_octal_like_escape_sequence(ch);
+            else
+                return read_octal_escape_sequence(ch);
+        }
```

Comment:
Given that at least Node.js accepts `"\08"` even under `"use strict"`, I don't think we should narrow the acceptable input range of `lib/parse.js` at all.